tmail-backend/deployment-tests/common/src/main/java/com/linagora/tmail/deployment/JmapContract.java

```diff
@@ -68,4 +78,35 @@ default void mailboxGetTest() {
            .body(ARGUMENTS + ".list.name", hasItems(expectedMailboxes.toArray()));
    }
 
+   @Test
+   default void sessionRouteShouldReturnDelegatedAccounts() {
+       RequestSpecification webadminRequestSpec = new RequestSpecBuilder()
+           .setContentType(ContentType.JSON)
+           .setAccept(ContentType.JSON)
+           .setConfig(newConfig().encoderConfig(encoderConfig().defaultContentCharset(StandardCharsets.UTF_8)))
+           .setBaseUri("http://" + jmapContainer().getContainerIpAddress())
+           .setPort(jmapContainer().getMappedPort(8000))
+           .setAuth(new NoAuthScheme())
+           .build();
+
+       // ALICE delegates BOB
+       given(webadminRequestSpec)
+       .when()
+           .put(String.format("/users/%s/authorizedUsers/%s", ALICE.asString(), BOB.asString()))
+       .then()
+           .statusCode(200);
+
+       // BOB's JMAP session should return BOB and ALICE accounts
+       String response = given()
+       .when()
+           .get("/jmap/session")
+       .then()
+           .statusCode(200)
+           .extract()
+           .body()
+           .asString();
+
+       assertThat(response).contains(BOB.asString(), ALICE.asString());
+   }
```

Comment:
Can we add tests for checking upload and download routes as well? (can be both in same test)

Reply:
Why do you want tests for upload and download routes? Is session routes test enough?

Reply:
All routes to be sure we don't have issues, it's in the description of the ticket